### PR TITLE
Jigsaw Puzzle Component

### PR DESCRIPTION
**Pull request by vivianliu0**
Loom Demo Video: 
https://www.loom.com/share/effd2dac6f2d43eea1ae5000f6de3045

### CODE REVIEW

**Review by ahmedhamodi — COMMENTED** [review]
Purely from a design perspective, I think it would look better if there was a gap between the grid positions, thoughts @aaronabraham311?

**Comment by vivianliu0** [comment]
Updated the spacing of the jigsaw puzzles.
Capture

File: samples/jigsaw_puzzle/jigsaw-prototype.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import cv2
```
Comment (jaydhulia) [inline samples/jigsaw_puzzle/jigsaw-prototype.py:1]: Note: to run this, need to install extra lib `pip install opencv-python`

**Review by jaydhulia — APPROVED** [review]
Nice work @vivianliu0, looks great to me!

File: samples/jigsaw_puzzle/jigsaw-prototype.py (hunk 1 above)
Comment (vivianliu0) [reply samples/jigsaw_puzzle/jigsaw-prototype.py:1]: Added a comment in the python file

**Review by jaydhulia — APPROVED** [review]

**Pull request merged**